python/ray/experimental/workflow/storage/__init__.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from ray.experimental.workflow.storage.base import Storage
+from ray.experimental.workflow.storage.filesystem import FilesystemStorageImpl
+
+import ray._private.utils
+
+
+def create_storage(storage_url: str) -> Storage:
+    """A factory function that creates different type of storage according
+    to the URL.
+
+    Args:
+        storage_url: A URL indicates the storage type and root path.
+
+    Returns:
+        A storage instance.
+    """
+    # TODO(suquark): in the future we need to support general URLs for
+    # different storages, e.g. "s3://xxxx/xxx". Currently we just use
+    # 'pathlib.Path' for convenience.
+    return FilesystemStorageImpl(storage_url)
+
+
+# the default storage is a local filesystem storage
+_global_storage = create_storage(
+    os.path.join(ray._private.utils.get_ray_temp_dir(), "workflows"))
```

Comment:
Please change this to use CWD by default. The case here is clear cut:
1. Temp dir is erased on system restart.
2. New users will have no idea where the data is stored otherwise! Placing it in cwd removes this source of user confusion right away.